cluster-autoscaler/simulator/cluster_snapshot_test.go

```diff
@@ -464,3 +463,126 @@ func TestNodeAlreadyExists(t *testing.T) {
 		}
 	}
 }
+
+func TestPVCUsedByPods(t *testing.T) {
```

Comment:
I'm not going to push super hard, but I feel this is a pretty minimal test coverage for a significant change in such a critical part of CA. For example forking or clearing are pretty big and critical parts of snapshot and I'd like to see tests that validate, for example, that Clear() also clears PVC data.

Reply:
Done.